tools/depends/target/config.site.in

```diff
@@ -103,9 +92,12 @@ if test "@platform_os@" = "darwin_embedded"; then
   fi
 fi
 
-# Python3 build for appletvos
 if test "@target_platform@" = "appletvos"; then
+  # Python3 build
   ac_cv_func_sigaltstack=no
+  # gnutls build
```

Comment:
Why add the lib info? This info might get outdated if libraries are updated.
What useful information does this comment provide?

Reply:
Easy to remove.